gratipay/utils/sentry.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+import traceback
+
+from aspen import log
+from gratipay import wireup
```

Comment:
> No neater way?

Not on this PR. 😞 

I agree it's hacky. Eventually I would see rewiring Sentry along the lines of what was started in #4345 and other PRs listed under" new email subflooring" on #4427.